Defs/Stats/Stats_Pawns_Combat.xml

```diff
@@ -3,6 +3,63 @@
 
   <!-- ================================== Shooting =======================================-->
 
+  <StatDef>
+    <defName>AimingDelayFactor</defName>
+    <label>aiming time</label>
+    <description>How long it takes to shoot after choosing a target.</description>
+    <category>PawnCombat</category>
+    <displayPriorityInCategory>4049</displayPriorityInCategory>
+    <defaultBaseValue>1</defaultBaseValue>
+    <minValue>0.5</minValue>
+    <toStringStyle>PercentZero</toStringStyle>
+    <showOnAnimals>false</showOnAnimals>
+    <skillNeedFactors>
+      <li Class="SkillNeed_Direct">
+        <skill>Shooting</skill>
+        <valuesPerLevel>
+          <li>1.25</li>
+          <li>1.2</li>
+          <li>1.17</li>
+          <li>1.15</li>
+          <li>1.12</li>
+          <li>1.09</li>
+          <li>1.06</li>
+          <li>1.03</li>
+          <li>1.0</li>
+          <li>0.98</li>
+          <li>0.96</li>
+          <li>0.94</li>
+          <li>0.92</li>
+          <li>0.90</li>
+          <li>0.88</li> 
+          <li>0.86</li>
+          <li>0.84</li>
+          <li>0.82</li>
+          <li>0.80</li> 
+          <li>0.78</li> 
+          <li>0.75</li>
+        </valuesPerLevel>
+      </li>
+    </skillNeedFactors>
+    <capacityFactors>
+      <li>
+        <capacity>Consciousness</capacity>
```

Comment:
Consciousness already affects Manipulation (among other things), so rather than having both capacities affect aiming time, I'd probably pick one or the other. My inclination would be to remove Consciousness--unless you strongly feel that it should be affected by the same capacities as AimingAccuracy for some reason.